db/re/item_trade.txt

```diff
@@ -712,14 +831,10 @@
 6402,499,100	// Oil_Palm_F
 6411,499,100	// Ripe_Watermelon
 6412,507,100	// Special_Medal
+6413,475,100	// New_Insurance
 6415,507,100	// Strange_Embryo
 6416,499,100	// Pet_Exchange
 6417,475,100	// Silvervine
-6418,499,100	// Agrade_Coin
-6419,499,100	// Bgrade_Coin
-6420,499,100	// Cgrade_Coin
-6421,499,100	// Dgrade_Coin
-6422,499,100	// Egrade_Coin
```

Comment:
Note: the **Xgrade_Coin** are account bound according to the item description on KRO sakray only.